ext/DynamicHMCExt.jl

```diff
@@ -0,0 +1,99 @@
+module DynamicHMCExt
+###
+### DynamicHMC backend - https://github.com/tpapp/DynamicHMC.jl
+###
+
+
+if isdefined(Base, :get_extension)
+    using Turing
+    import DynamicHMC
+    import AbstractMCMC
+    import Random
+    import LogDensityProblems
+    import LogDensityProblemsAD
```

Comment:
BTW this is not OK - extensions should only load the main package (Turing) and the weak dependency (DynamicHMC). Everything else (including stdlibs) has to be loaded via those or made an additional weak dependency.

Reply:
Ok let me know if the current approach is better.

Reply:
Seems good :slightly_smiling_face: